### PR TITLE
Add Google Presence

### PR DESCRIPTION
**Pull request by TheGamerzs**

### CODE REVIEW

File: src/Presence/google.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+		"description": "Title for google books"
+	},
+	"google.doodles": {
+		"message": "Doodles",
+		"description": "Title for google doodles"
+	},
+	"google.about": {
+		"message": "About",
+		"description": "Title for google about"
+	},
```
Comment (Bas950) [inline src/Presence/google.json:13]: context?

File: src/Presence/google.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	"google.home": {
+		"message": "Home",
+		"description": "Title for google Home"
+	},
```
Comment (Bas950) [inline src/Presence/google.json:17]: general.viewHome ?

File: src/Presence/google.json (hunk 1 above)
Comment (TheGamerzs) [reply src/Presence/google.json:13]: Doodles about page

File: src/Presence/google.json (hunk 2 above)
Comment (TheGamerzs) [reply src/Presence/google.json:17]: no?

File: src/Presence/google.json (hunk 1 above)
Comment (Bas950) [reply src/Presence/google.json:13]: well update the description to make it more clear...

File: src/Presence/google.json (hunk 2 above)
Comment (Bas950) [reply src/Presence/google.json:17]: yes?

File: src/Presence/google.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+		"description": "Title for google doodles"
+	},
+	"google.viewAbout": {
+		"message": "About",
+		"description": "Title for google doodles about page"
+	},
+	"google.viewHome": {
+		"message": "Home",
+		"description": "Title for google home"
+	},
```
Comment (Bas950) [inline src/Presence/google.json:17]: home what?
Google Home: https://play.google.com/store/apps/details?id=com.google.android.apps.chromecast.app&hl=nl&gl=US
Or Viewing the home page? (search engine)

Comment (TheGamerzs) [reply src/Presence/google.json:17]: Home Page

Comment (Bas950) [reply src/Presence/google.json:17]: yea so make that more clear....

**Pull request closed without merging**